CHANGELOG.md

```diff
@@ -16,18 +16,24 @@ Remember to align the itemized text with the first line of an item within a list
     * `random.rbg_key(seed)` becomes `random.PRNGKey(seed, impl='rbg')`
     * `random.unsafe_rbg_key(seed)` becomes `random.PRNGKey(seed, impl='unsafe_rbg')`
 * Changes:
-  * {func}`jax.scipy.stats.mode` now returns a 0 count if the mode is taken
-    across a size-0 axis, matching the behavior of `scipy.stats.mode` in SciPy
-    1.11.
+  * CUDA: JAX now verifies that the CUDA libraries it finds are at least as new
```

Comment:
So this will break things if we ship cuda 12.2 and users have 12.1?

Reply:
Correct. JAX will raise an exception if you do this.

Reply:
What was the reason? (sorry I don't have the context nor did I know about this)

Reply:
I'm fairly confident this is responsible for a number of user issue where too old a  version is chosen and mysterious failures happen. I'd much rather we caught this early and loudly.

Reply:
Ok